added posts and comments interaction

maisykiwi opened the pull request.

maisykiwi committed:
added posts and comments interaction

lucid281 commented on bots/jsonplaceholder.py:
```diff
@@ -114,6 +116,89 @@ def get():
+            setattr(self, attr, post_json[attr])
+
+        # assign methods with names from bot.endpoints to a generated function
+        for endpoint in ["posts", "comments", "comments", "albums", "todos"]:
```
not sure if these are corrects for 'posts'

maisykiwi committed:
updated endpoints for users, comments, posts

lucid281 committed:
fixed call to posts
previous code was expecting a body parameter for the id and for the call to return a list
new code fixes parameter to be in url and returns a hash

lucid281 committed:
Merge remote-tracking branch 'origin/add_posts_n_comments' into add_posts_n_comments

lucid281 commented on bots/jsonplaceholder.py:
```diff
@@ -114,6 +116,90 @@ def get():
+
+    def by_id(self, id):
+        """Get a Post object by id"""
+        post_json = self._b.r.get(f'{self._b.endpoints["posts"]}/{id}').json()
```
this is the line i had to fix

lucid281 replied:
changed from taking the id parameter in the body, to taking it in the url.
